docs/guide/quickstart.rst

```diff
@@ -0,0 +1,298 @@
+    .. image:: /images/discord_create_app_button.png
+        :scale: 90 %
+
+3. Navigate over to the ``bot`` tab:
+    .. image:: /images/discord_bot_tab.png
+        :scale: 80%
+
+4. Create a new Bot user if you have not done so already:
+    .. image:: /images/discord_create_bot_user.png
+
+5. If you need these `intents <_guide_intents>`_, scroll down and click on these check boxes:
+    .. image:: /images/discord_privileged_intents.png
+
+6. Your bot token will only be retrievable upon creation and looks like this:
+    .. image:: /images/discord_bot_token.png
+
+6a. Otherwise it will look like this:
+    .. image:: /images/discord_bot_token_no_copy.png
+
+.. danger::
+
+    This token is **VERY IMPORTANT** and you should treat it like the password to your bot. Keep it as secure as possible and never give this to anyone.
+    If you ever leak it, you can click the ``Regenerate`` button to forcibly invalidate all copies of the token. This will also terminate all running instances of your bot.
+
+.. _guide_quickstart_client_secret:
+
+.. warning::
+
+    You may be thinking of using the ``Client Secret`` for your bot token. This is **NOT** what you are looking for - you need the bot **Token**, which is a completely different format.
+
+    .. image:: /images/discord_client_secret_big_nono.png
+        :scale: 90 %
+
+A Simple Bot
+-------------
+
+This step-by-step walk-through will show you how to make a bot using the commands framework.
+
+.. note::
+
+    This walk-through does not cover application commands (slash commands). For a detailed walk-through of application commands, see _guide_app_commands.
+
+
+1. Create a new Python file in the folder you want to work in.
+
+    .. warning::
+
+        Do not name the file ``discord.py``, as this will cause conflicts with the discord.py library. Also, do not create a sub-folder in your project named ``discord``, as that too will cause conflicts.
+
+2. Open the new Python file in your preferred editor.
+
+    If you do not have an editor installed, you can use a community recommended one, such as `Visual Studio Code <https://code.visualstudio.com/>`_,
+    `PyCharm <https://www.jetbrains.com/pycharm/>`_ or `Sublime Text 4 <https://www.sublimetext.com/>`_.
+    We don't recommend Python's built in IDLE, as the lack of features compared to other simple editors makes it very bothersome for projects with many files.
+
+3. Now you can start creating your bot. The following steps will go over a simple bot line-by-line to help you understand what's happening.
+
+First, you need to import the discord.py library:
+
+.. code-block:: python3
+
+    import discord
+    from discord.ext import commands
+
+We import ``discord.ext.commands`` here as we will need it for our bot.
+
+.. code-block:: python3
+
+    intents = discord.Intents.default()
+    intents.message_content = True
+    bot = commands.Bot(command_prefix="!", description="This is my awesome bot!", intents=intents)
```

Comment:
```suggestion
    bot = commands.Bot(command_prefix='!', description='This is my awesome bot!', intents=intents)
```

We use single quotes throughout the documentation and code. That being said, generally subclassing is preferred nowadays since almost all the examples do it.